### DISCUSSION TITLE
Institution name and domain address @iacs.res.in

### DISCUSSION THREAD

**Discussion by meghacy** [RosettaCommons/rosetta#108] [ROSIE]
Dear ROSIE Support Team,

I am writing to inform you that my email address with the domain "@iacs.res.in" belongs to a research institute called "IACS" (Indian Association for the Cultivation of Science), which is located in Kolkata, West Bengal, India

I would appreciate it if you could add my email address to your system to grant me access to the server.
Thank you for your attention to this matter.

Best regards,
Megha Agarwal
Senior Research Fellow, School of Chemical Sciences
IACS,
Kolkata-700032
West Bengal
India

**Comment by lyskov** [comment]
Thank you for letting us know, - this should be fixed now. Please feel free to re-open if you still having issues.
